src/client/javascripts/file-upload-spinner.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const form = document.querySelector('form')
+  const uploadSpinner = document.querySelector('form ~ .apha_upload-spinner')
```

Comment:
Is this a bit fragile?  Would expect we could put an id on the spinner & address it directly.